.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,73 +1,16 @@
-## Status
+Fixes #
 
-Ready for review / Work in progress
+## Description
```

Comment:
Is having the "Description" header at the top useful? I mainly ask because GitHub will copy your commit message to the top of the PR, so if we keep the heading, you then have to cut and paste the commit message from above the header to below it. 

I'm also okay with having "Fixes X" be after the commit message, since that's where it already goes in the commit message, and the GitHub UI now highlights it in the sidebar too.